### DISCUSSION TITLE
possible conflicts with crypto lib, and a general query.

### DISCUSSION THREAD

**Discussion by DeveloperKev** [Homebrew/discussions#3577] [Tap maintenance and Homebrew development]
what is `cellar` directory? can i change it from `/opt/homebrew/Cellar` to local?

Another thing, https://github.com/wbond/package_control/issues/1612
brew is conflicting with the `crypto library`, pls resolve it.

**Comment by SMillerDev** [comment]
As I answered in https://github.com/orgs/Homebrew/discussions/3571#discussioncomment-3498571

> brew is conflicting with the crypto library, pls resolve it.

That's not what's in that issue

> what is cellar directory?

https://docs.brew.sh/Formula-Cookbook#homebrew-terminology

> can i change it from /opt/homebrew/Cellar to local?

I don't know what local is supposed to be in this context, but if you change the cellar location all your Homebrew software is going to break so this is a very bad idea.
